tests/test_ride.py

```diff
@@ -192,15 +190,13 @@ def test_already_created_ride_offer(self):
         """
             Test a user cannot create more than one ride at a time.
         """
-        response = self.client().post('/api/v1/auth/signup', data=self.user_data_2)
-        self.assertEqual(response.status_code, 201)
+        self.client().post('/api/v1/auth/signup', data=self.user_data_2)
         login_response = self.client().post('/api/v1/auth/login', data=self.user_data_2)
-        self.assertEqual(login_response.status_code, 200)
         #Define header dictionary
         access_token = json.loads(login_response.data.decode())['access_token']
         #create a new ride offer
-        response = self.client().post('/api/v1/ride/create',
-                                      headers=dict(Authorization='Bearer ' + access_token), data=self.ride_data_1)
+        self.client().post('/api/v1/ride/create',
+                           headers=dict(Authorization='Bearer ' + access_token), data=self.ride_data_1)
```

Comment:
You can set the authorization header on the setup method instead of having to set it here.

Reply:
Thanks let me try that